stack-citationapi/recipes/deploy-publicapi.rb

```diff
@@ -0,0 +1,32 @@
+node['deploy'].each do |application, deploy|
```

Comment:
While you are at it: Can you rename this to `deploy-citation-formatting-api.rb`?